rasterio/features.py

```diff
@@ -155,6 +155,7 @@ def rasterize(
         out=None,
         transform=IDENTITY,
         all_touched=False,
+        merge_alg_add=False,
```

Comment:
I think this should be `merge_alg='replace'`. In theory, there could be other merge algorithms added to GDAL in the future, and we'd be forward compatible.

Reply:
A corresponding enum like 

```python
class MergeAlg(Enum):
    """Available rasterization algorithms"""
    replace = 'REPLACE'
    add = 'ADD'
```

could help: we could use this to validate inputs.